test/unit/datasets/DatasetsRepository.test.ts

```diff
@@ -1016,6 +1017,38 @@ describe('DatasetsRepository', () => {
     })
   })
 
+  describe('getDatasetDownloaCount', () => {
+    const testDatasetDownloadCount: DatasetDownloadCount = {
+      id: testDatasetModel.id,
+      downloadCount: 1,
+      MDCStartDate: '2021-01-01'
+    }
+    test('should return citation when response is successful', async () => {
```

Comment:
update test description
```suggestion
    test('should return download count when response is successful', async () => {
```